ValveResourceFormat/CompiledShader/ShaderFile.cs

```diff
@@ -571,7 +571,6 @@ public byte[] GetDecompressedZFrame()
                         {
                             throw new ShaderParserException("Decompressed zframe doesn't match expected size");
                         }
-                        zstdDecoder.Dispose();
```

Comment:
Again crashing when multiple threads try to load a zframe. Something with the dictionary.

Reply:
Maybe it's the parser not knowing where the end of the file is

Reply:
It was a thread safety issue fixed here https://github.com/SteamDatabase/ValveResourceFormat/commit/09598e954d322227fdd09386fb13b9817b874b62